service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapper.java

```diff
@@ -68,6 +70,11 @@ public String mapObservationToObservationStatement(Observation observation, bool
             }
         }
 
+        var bodySite = extractTextOrCoding(observation.getBodySite())
```

Comment:
IS it worth extracting this into a small sub function if the code is the same in both cases here?